go/flags/endtoend/vttablet.txt

```diff
@@ -256,6 +256,8 @@ Usage of vttablet:
       --restore_concurrency int                                          (init restore parameter) how many concurrent files to restore at once (default 4)
       --restore_from_backup                                              (init restore parameter) will check BackupStorage for a recent backup at startup and start there
       --restore_from_backup_ts string                                    (init restore parameter) if set, restore the latest backup taken at or before this timestamp. Example: '2021-04-29.133050'
+      --restore_to_pos string                                            (init incremental restore parameter) if set, run a point in time recovery that ends with the given position. This will attempt to use one full backup followed by zero or more incremental backups
+      --restore_to_timestamp 2006-01-02T15:04:05Z07:00                   (init incremental restore parameter) if set, run a point in time recovery that restores up to the given timestamp, if possible. Given timestamp in RFC3339 format (2006-01-02T15:04:05Z07:00)
```

Comment:
Why is there a date next to the flag name? The type should be timestamp (or something like that).
Also in a later PR, we should explore whether to combine this flag with ` --restore_from_backup_ts string ` if possible.

Reply:
Oh, actually no idea why `--restore_to_timestamp 2006-01-02T15:04:05Z07:00`!

Reply:
Fixed.

Reply:
> combining with --restore_from_backup_ts is an option, but might be confusing because the flag already has a different meaning

Yes, that's why I've deferred the discussion on it 😅